sdk/src/main/java/org/coralibre/android/sdk/fakegms/tasks/ResultTask.java

```diff
@@ -0,0 +1,34 @@
+package org.coralibre.android.sdk.fakegms.tasks;
+
+import androidx.annotation.NonNull;
+
+final class ResultTask<T> extends Task<T> {
```

Comment:
Maybe add a short comment (javadoc of this class) that tells that this type of Task is always completed with success, so that one does not have to read the code to understand this.